Tests/FluentAssertions.Specs/Extensions/FluentActionsSpecs.cs

```diff
@@ -51,6 +52,36 @@ public void Enumerating_works_with_specialized()
             Enumerating(() => ThrowsAfterFirst(0)).Should().Throw<InvalidOperationException>();
         }
 
+        [Fact]
+        public void Enumerating_works_with_enumerable_func()
+        {
+            // Arrange
+            var testee = new Example();
```

Comment:
We usually use `subject` or `actual` for the SUT, please use one of those.
It's not that `testee` in any way is nonsensical, but just to avoid introducing yet another naming.